files/galaxy/tpv/destinations.yml.j2

```
@@ -233,6 +233,32 @@ destinations:
       require:
         - cz-pulsar
 
+  pulsar_cz02_tpv:
+    inherits: pulsar_default
+    runner: pulsar_eu_cz02
+    max_accepted_cores: 32
+    max_accepted_mem: 128
+    min_accepted_gpus: 1
+    max_accepted_gpus: 1
+    env:
+      LC_ALL: C
+      SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      SINGULARITY_TMPDIR: "/storage/praha5-elixir/home/galaxyeu/singularity/tmp"
+      TMPDIR: "$SCRATCHDIR"
+      TMP: "$SCRATCHDIR"
+      TEMP: "$SCRATCHDIR"
+      ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.3.1"
+      NGPUS: "$PBS_NGPUS"
+      GPU_AVAILABLE: 1
```

I think it does not matter, but since `"1"` and `1` have different meanings in YAML, and [we are using `"1"`](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/468b63fcb70bf70637f244991706a99034dbb6f1/files/galaxy/tpv/destinations.yml.j2#L77), I would advise you to change it just because it is easy and can prevent unpleasant surprises.

```suggestion
      GPU_AVAILABLE: "1"
```